src/main/java/uk/gov/hmcts/reform/mi/miextractionservice/component/impl/CoreCaseDataExportBlobDataComponentImpl.java

```diff
@@ -93,27 +92,28 @@ public String exportBlobsAndGetOutputName(BlobServiceClient sourceBlobServiceCli
 
         boolean dataFound = readAndWriteDataAsCsv(sourceBlobServiceClient, fromDate, toDate);
 
-        if (Boolean.FALSE.equals(dataFound)) {
-            throw new ExportException("No data to output.");
-        }
+        if (dataFound) {
+            encryptArchiveComponent.createEncryptedArchive(Collections.singletonList(CCD_WORKING_FILE_NAME), CCD_WORKING_ARCHIVE);
 
-        encryptArchiveComponent.createEncryptedArchive(Collections.singletonList(CCD_WORKING_FILE_NAME), CCD_WORKING_ARCHIVE);
+            BlobContainerClient blobContainerClient = targetBlobServiceClient.getBlobContainerClient(CCD_OUTPUT_CONTAINER_NAME);
 
-        BlobContainerClient blobContainerClient = targetBlobServiceClient.getBlobContainerClient(CCD_OUTPUT_CONTAINER_NAME);
+            if (!blobContainerClient.exists()) {
+                blobContainerClient.create();
+            }
 
-        if (!blobContainerClient.exists()) {
-            blobContainerClient.create();
-        }
+            String outputBlobName = fromDate.format(dateTimeUtil.getDateFormat())
+                + NAME_DELIMITER
+                + toDate.format(dateTimeUtil.getDateFormat())
+                + NAME_DELIMITER
+                + CCD_WORKING_ARCHIVE;
 
-        String outputBlobName = fromDate.format(dateTimeUtil.getDateFormat())
-            + NAME_DELIMITER
-            + toDate.format(dateTimeUtil.getDateFormat())
-            + NAME_DELIMITER
-            + CCD_WORKING_ARCHIVE;
+            blobContainerClient.getBlobClient(outputBlobName).uploadFromFile(CCD_WORKING_ARCHIVE, true);
 
-        blobContainerClient.getBlobClient(outputBlobName).uploadFromFile(CCD_WORKING_ARCHIVE, true);
+            return outputBlobName;
+        }
 
-        return outputBlobName;
+        log.info("No data found to output.");
```

Comment:
maybe log.error? It should be data every week

Reply:
I think log.warn then? Since I feel error should be breaking?